### DISCUSSION TITLE
Kolibri Entwicklung (Storybook, Dokumentation, Komponentenübersicht)

### DISCUSSION THREAD

**Discussion by bastiW** [public-ui/kolibri#6008] [Q&A] [answered]
Hi, 

ich habe ein paar Fragen zum Entwickeln mit Kolibri, worüber ich keine Informationen in der Readme oder Dokumentation finden konnte:

1. Wie lässt sich das Integrierte Storybook lokal starten? Für was wird dies verwendet?
2. Wie lässt sich lokal die Dokumentation wie auf der Website starten https://public-ui.github.io/docs/components
3. Ich habe gesehen, dass es über die [Deployte Umgebung ](https://kolibri-public-ui.netlify.app/) eine Ansicht gibt, die alle Komponenten in eine Navigation packt und dann anzeigt. Ich habe nur diese übersicht mittels dem "start" Command im "components" Folder gefunden. Wie lässt sich die in netlify gefundene übersicht lokal starten?

<img width="1272" alt="image" src="https://github.com/public-ui/kolibri/assets/7667003/0c7c0647-0bf0-4982-8b5b-c5705af80c7d">

**Comment by bastiW** [comment]
zu 3)  befindet sich im Samples React Projekt. Readme in packages/samples/react/README.md

**Comment by deleonio** [comment, marked as answer]
Hallo @bastiW,

vielen Dank für Dein Interesse. Hier ein paar Antworten zu Deinen Fragen:

1. Wir verwenden Docusaurus statt Storybook (canceled). Das Repo befindet sich in einem andere  Repo: https://github.com/public-ui/public-ui.github.io

3. Die Sample-App ist das React-Beispiel. Diese kann einfach mit `npm start` gestartet werden: https://github.com/public-ui/kolibri/tree/develop/packages/samples/react
